ansible/playbook/README.md

```diff
@@ -1,4 +1,98 @@
 
+# Introduction
+
+This document describes the Ansible implementation in the Snowdrop project.
+
+When in this document it refers to the `controller` it means the machine that executes the Ansible playbooks and roles and that owns the inventory, *i.e.*, the 
+machine that clones this project.
+
+The `hosts` are the machines where the playbooks and roles are executed against. 
+
+The exception goes to the playbooks that are executed against `localhost`. This special host is referred to the `controller`. 
+
+# Ansible Inventory
+
+Ansible inventory manages access to hosts using Ansible.
```

Comment:
Is this sentence correct - `Ansible inventory manages access to hosts using Ansible` as it looks like for an ellipse ?

Reply:
I improved the description.

Reply:
Done.